gravis3d/play.py

```diff
@@ -14,4 +14,4 @@
 sleep(wait)
 while(1):
     rate = R
-    universe.update()
+    universe.update()
```

Comment:
What change was done here?

Reply:
nothing i think i might added something here but later on found not intresting in that.